ntbs-service/Pages/Shared/_NotificationBanner.cshtml

```diff
@@ -1,79 +1,25 @@
-@using ntbs_service.Models.Enums;
 @{

```

Comment:
I extracted the actual notification display bit into a partial, as the same design is used to show the linked notifications

Reply:
Sure, I'll make sure I take care when merging :)